.github/workflows/ci.yml

```
@@ -17,16 +17,25 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version:
-          - 5.3
-          - 5.4
-          - 5.5
-          - 5.6
-          - 7.0
-          - 7.1
-          - 7.2
-          - 7.3
-          - 7.4
+        include:
+          - php-version: 5.3
+            coverage: none
+          - php-version: 5.4
+            coverage: none
+          - php-version: 5.5
+            coverage: none
+          - php-version: 5.6
+            coverage: none
+          - php-version: 7.0
+            coverage: none
+          - php-version: 7.1
+            coverage: none
+          - php-version: 7.2
+            coverage: none
+          - php-version: 7.3
+            coverage: none
+          - php-version: 7.4
+            coverage: xdebug
```

This could be made simpler with:

```suggestion
        php-version: ['5.3', '5.4', '5.5', '5.6', '7.0', '7.1', '7.2', '7.3']
        coverage: ['none']
        include:
          - php-version: 7.4
            coverage: xdebug
```

---

Thanks, @pierlon!

---

@oliverklee is there a reason you haven’t followed @pierlon’s suggestion?

---

I think I included it. Possibly it got lost when I squashed or rebased. I'll update this PR and repush.

---

Fixed and repushed.